Hw8

mojjevelovii opened the pull request.

mojjevelovii committed:
сделала получение данных о текущей погоде в городе
получение данных о погоде на неделю
вынесла работу с сетью в отдельный класс

mojjevelovii committed:
очистка неиспользуемых ресурсов

DwarfPorter commented on app/src/main/java/ru/shumilova/weatherapp/data/WeatherRepository.java:
```diff
@@ -0,0 +1,142 @@
+    public void getWeatherWeek (final String cityName) {
+        new Thread(new Runnable() {
+            @Override
+            public void run() {
```
Тут практически тот же код, что и в предыдущем методе. Можно ли как-то их объединить?

DwarfPorter commented on app/src/main/java/ru/shumilova/weatherapp/data/WeatherRepository.java:
```diff
@@ -0,0 +1,142 @@
+                try {
+                    url = new URL(urlString);
+                    urlConnection = (HttpURLConnection) url.openConnection();
+                    urlConnection.addRequestProperty("x-api-key", "cfdb77e65d320ca11996d5fee7f947bd");
```
Ключ бы вынести из репозитория

mojjevelovii committed:
Выполнены правки по комментариям преподавателя:
схожий код объединен в отдельный метод
ключ вынесен из репозитория в проперти (проперти загружаю в гит для возможности последующей проверки)
